nautobot_chatops/dispatchers/ms_teams.py

```diff
@@ -85,10 +86,21 @@ def _send(self, content, content_type="message"):
                 "replyToId": self.context["message_id"],
             }
         )
+        json_content = json.dumps(content)
+        max_msg_size = settings.PLUGINS_CONFIG["nautobot_chatops"]["microsoft_max_msg_size"]
+        if 0 < max_msg_size < len(json_content):
+            msg = "Prepared content, size %s, exceeded maximum allowed MS Teams msg size." % len(json_content)
+            logger.warning(msg)
+            json_content = json.dumps(
+                self.send_error(
```

Comment:
I don't think I like how this recurses - in `_send`, we're now calling `send_error`, which calls `send_blocks`, which calls `_send`. Won't that result in a duplicate message being sent in the best case, and infinite recursion in the worst case (where `max_msg_size < len(json.dumps(self.send_error(...)))`)?

I think it'd be better and safer to just construct the block directly here, i.e.:
```python
json_content = json.dumps([{
    "type": "TextBlock",
    "text": "Message size exceeds maximum allowed by MS Teams.\nConsider changing the filtering method.",
    "wrap": True,
    "color": "attention",
}])
```

Reply:
That's a good point, constructing the block directly would be much safer.